### ISSUE TITLE
Create Firefox extension

### ISSUE THREAD

**Issue by tunnckoCore** [domeccleston/sharegpt#14]
You: Can you convert simple and basic Chrome extension into a firefox addon?
Chat GPT: Yes, it is possible to convert a simple and basic Chrome extension into a Firefox addon. To do this, you will need to follow these steps:

1. Open your Chrome extension in a text editor and locate the manifest file. This file should be named `manifest.json` and contains information about your extension, such as its name and version number.

2. Open the manifest file and modify the `"browser_specific_settings"` section to include the following code:

```
"browser_specific_settings": {
    "gecko": {
        "id": "your-addon-id-on-AMO",
        "strict_min_version": "48.0"
    }
},
```

3. Save the changes to the manifest file.

4. Use a tool such as the Web Extension Polyfill to convert the Chrome-specific APIs used in your extension to their Firefox equivalents.

5. Test your extension in Firefox to make sure it works as expected.

6. Submit your extension to the Firefox Add-ons website for review and publishing.

Keep in mind that the process of converting a Chrome extension to a Firefox addon can be more complex for extensions that use advanced features or APIs. It is always a good idea to consult the Firefox Add-ons Developer documentation for guidance and support.

**Comment by tunnckoCore** [comment]
There are no specific APIs, so I guess it's just that `manifest` option, and upload it to Firefox Addons.

**Comment by punchagan** [comment]
FWIW, I was able to install this extension in Firefox by cloning the repo and installing the addon from the cloned directory.

**Comment by tunnckoCore** [comment]
OKay, thanks, thought about that too.

**Comment by kojix2** [comment]
@hidao80 created a firefox extension for ShareGPT
https://github.com/domeccleston/sharegpt/issues/52

**Comment by hsiangfeng** [comment]
> @hidao80為 ShareGPT 創建了一個 firefox 擴展 [#52 ](https://github.com/domeccleston/sharegpt/issues/52)

It can no longer be used now.

**Comment by hidao80** [comment]
I have released version v0.4, which addresses the URL and DOM tree changes associated with the upgrade of ChatGPT.
If you would like, please feel free to use this new version. #52
